modules/git/command.go

```diff
@@ -153,6 +153,7 @@ func (c *Command) RunInDirTimeoutEnvFullPipelineFunc(env []string, timeout time.
 		err := fn(ctx, cancel)
 		if err != nil {
 			cancel()
+			_ = cmd.Wait()
```

Comment:
Should we at least log if there is an error here?

Reply:
There will always be an error here - at least the ctx deadline exceeded error - if not Stdin or Stdout closed or something else like that. Any that will terminate the git command. 

The user of the function has determined what error they want to return `err` . Any other error is because of that.